css-components/components-src/stylus/components/list.styl

```diff
@@ -256,12 +256,9 @@ list__item()
 .list__item--tappable
   transition background-color 0.2s linear
 
-.list__item--tappable:active
+.list__item--tappable-active
```

Comment:
It looks like the modifier is called "tappable-active". Can you change to `.list__item--tappable--active`?